modules/ocf_prometheus/manifests/alertmanager.pp

```diff
@@ -0,0 +1,40 @@
+class ocf_prometheus::alertmanager {
+  ocf::firewall::firewall46 {
+    '995 allow prometheus alertmanager to send on SMTP port':
```

Comment:
We may want to consider consolidating this rule and the one in `ocf_jenkins`.

Reply:
Is it really necessary to add more layers of abstraction here? I don't think this repetition is so bad, especially since it's clear what's happening here.